test/modules/linear_boltzmann_solvers/transport_steady/transport_2d_triangular_P0.lua

```diff
@@ -0,0 +1,126 @@
+-- 2D LinearBSolver Vacuum with isotropic incident, triangular set
+-- SDM: PWLD
+-- Test: 
+num_procs = 1
+
+--############################################### Check num_procs
+if (check_num_procs==nil and number_of_processes ~= num_procs) then
+    log.Log(LOG_0ERROR,"Incorrect amount of processors. " ..
+            "Expected "..tostring(num_procs)..
+            ". Pass check_num_procs=false to override if possible.")
+    os.exit(false)
+end
+
+--############################################### Setup mesh
+nodes = {}
+N = 65
+ds = 500.0 / N
+for i = 0, N do
+    nodes[i + 1] = i * ds
+end
+meshgen1 = mesh.OrthogonalMeshGenerator.Create({ node_sets = { nodes, nodes } })
+mesh.MeshGenerator.Execute(meshgen1)
+
+--############################################### Set Material IDs
+vol0 = logvol.RPPLogicalVolume.Create({ infx = true, infy = true, infz = true })
+mesh.SetMaterialIDFromLogicalVolume(vol0, 0)
+--############################################### Add materials
+materials = {}
+materials[1] = mat.AddMaterial("Test Material");
+
+mat.AddProperty(materials[1], TRANSPORT_XSECTIONS)
+
```

Comment:
delete line

Reply:
Extra lines were deleted, and cases condensed.